ios/Classes/AblyFlutterPlugin.m

```diff
@@ -39,6 +39,44 @@ -(nullable AblyFlutter *)ablyWithHandle:(NSNumber *)handle;
     [plugin registerWithCompletionHandler:result];
 };
 
+static FlutterHandler _createRestWithOptions = ^void(AblyFlutterPlugin *const plugin, FlutterMethodCall *const call, const FlutterResult result) {
+    AblyFlutterMessage *const message = call.arguments;
+    LOG(@"message for handle %@", message.handle);
```

Comment:
We need to improve this, yes. @tiholic is just working with a macro which [I put in place](https://github.com/ably/ably-flutter/commit/cddb67bcb1b9a91ed47ada7a53a751609e794318) early on in my experiments with Flutter.

I've created #7 where this can be discussed further.

Reply:
Resolution of that new issue doesn't need to block this PR from landing, to be clear.

Reply:
@QuintinWillison yes, the same macro being used here.